src/main/java/module_two_1/models/enums/DisciplineNames.java

```diff
@@ -0,0 +1,16 @@
+package module_two_1.models.enums;
+
+public enum DisciplineNames {
+
+    MATHS("математика"), HISTORY("история"), CULTURE("культура"),
+    RUSSIAN_LANGUAGE("русский язык"), ENGLISH("английский язык"), BASES_OF_LAW("основы права");
```

Comment:
I see you have enum values that are not used in your code. Is there a reason not to delete them?

Reply:
Deleted